Calculator/Calculator/Model/Formula.swift

```diff
@@ -7,7 +7,28 @@
 
 import Foundation
 
-struct Formula<T> {
-    var operands: CalculatorItemQueue<Double> = CalculatorItemQueue()
-    var operators: CalculatorItemQueue<Operator> = CalculatorItemQueue()
+
+struct Formula {
+    var operands: CalculatorItemQueue<Double>
+    var operators: CalculatorItemQueue<Operator>
+    
+    public mutating func result() throws -> Double {
+        guard var result = operands.dequeue() else {
+            throw CalculateError.unavailableDequeue
+        }
+        guard operands.count == operators.count else {
+            throw CalculateError.operandOperatorCountNotMatching
+        }
+        for _ in 1...operators.count {
```

Comment:
```suggestion
        for _ in 0 ..< operators.count -1 {
```
 결국에는 count 값 만큼 for 문을 돌리는 것이라 관례상 이 표현이 더 자연스러울 것 같아요 :)